### ISSUE TITLE
Wrong description of positive class weight in BCEWithLogitsLoss

### ISSUE THREAD

**Issue by siarez** [pytorch/pytorch#16804]
## 📚 Documentation

The explanation of `pos_weight` for [BCEWithLogitsLoss](https://pytorch.org/docs/stable/nn.html#torch.nn.BCEWithLogitsLoss) mentions:
`where :math:`p_n` is the positive weight of class :math:`n`.`

We don't have `n` classes (this is a binary loss). The correct description would be something like this:
 `:math:p_n is the weight of the positive class for sample n in the batch`

**Comment by velikodniy** [comment]
I agree, there is a typo but another one. Instead of `p_n` should be `p_c` where `p_c` is the positive weight of class `c`. It's a binary loss, but we can use it for training a multi-label (but not multi-class) classifier.

Actually it's useful because you can control precision/recall in imbalanced datasets with it.

**Comment by velikodniy** [comment]
I think we should explain it in the documentation.
